api/src/main/java/net/automatalib/AutomataLibProperty.java

```diff
@@ -43,6 +43,25 @@
      */
     LTSMIN_PATH("ltsmin.path"),
 
+    /**
+     * {@code ltsmin.check.version}.
+     * <p>
+     * Whether to check LTSmin versions, e.g. when working with a development version of LTSmin which has not yet
+     * been released you may want to set this property to "false".
+     * <p>
+     * If set to "false" the check will be skipped, if set to anything else the check will be done.
+     */
+    LTSMIN_CHECK_VERSION("ltsmin.check.version"),
```

Comment:
I even see a problem with current versions. I have an old v3.0.0 installation lying around. If I run a `ltsmin-convert --version` I get `do-not-publish-website` as a version. So even with an installation that supports the current (non-monitor) stuff, I would have to go the extra mile to figure out what the issue is and then reconfigure accordingly.

I think this just adds inconvenience, if in half of the cases one has to disable this. Maybe we should make the version check completely optional and just log a warning if we detect a mismatch? So in case some error occurs we can hint the user towards the issue.